rllib/core/rl_module/rl_module.py

```diff
@@ -315,3 +362,29 @@ def as_multi_agent(self) -> "MultiAgentRLModule":
         from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
 
         return MultiAgentRLModule({DEFAULT_POLICY_ID: self})
+
+    @staticmethod
+    def __check_if_args_kwargs_serializable(args, kwargs):
```

Comment:
dude, I feel like this should not be a static function on RLModule.
this is a very fundamental util that is worth living somewhere with a lot of unit tests of its own.

Reply:
like, this has very little to do with RLModule.

Reply:
I moved this function to rllib/utils/serialization.py